
 
National Academies of Sciences, Engineering, and Medicine; Division on Earth and Life Studies; Board on Life Sciences; Board on Environmental Studies and Toxicology; Committee on Advancing Understanding of the Implications of Environmental-Chemical Interactions with the Human Microbiome. Environmental Chemicals, the Human Microbiome, and Health Risk: A Research Strategy. Washington (DC): National Academies Press (US); 2017 Dec 29.
 
The human microbiome has evolved to be a remarkably diverse, finely balanced, and highly environment-specific ecosystem (Lloyd-Price et al. 2016). Each body site constitutes a specific habitat that can include trillions of microbial cells and hundreds of strains that differ nearly completely from one site to another throughout the body (HMP Consortium 2012a,b). Using techniques from molecular epidemiology, microbial ecology, and microbiology, researchers have demonstrated that changes in typical immune interactions, biomolecular activities, or pathogen exclusion are associated with such diseases as inflammatory bowel disease, autism, and cancer (Bckhed et al. 2012; Hsiao et al. 2013; Petersen and Round 2014; Trompette et al. 2014; Garrett 2015). Culture-independent population studies of the human microbiome follow an approach similar to that of Franzosa et al. (2015), which includes collecting stabilized microbial biomass specimens at various times from people who have various exposures or phenotypes of interest; assaying the collected samples with one or more molecular profiling technologies (Segata et al. 2013); bioinformatically profiling the resulting raw data to quantify microbial features of interest, such as species abundances, strains, and biomolecular functional elements; and statistically associating changes in those features with population phenotypes or exposures. That approach is similar to other types of molecular epidemiology studies, such as gene-expression biomarker discovery or genome-wide association studies, and can be combined with experimental approaches that change or challenge the microbiome.
 
**DOWNLOAD âš¹ [https://fienislile.blogspot.com/?download=2A0SPO](https://fienislile.blogspot.com/?download=2A0SPO)**


 
Because of the nature of human microbiome studies, the resulting associations are most often correlative rather than clearly causal; however, additional targeted assays can be used to establish causality and mechanism. The most common targeted assays might involve gnotobiotic transfer of human microbiome samples into controlled model organisms (such as mice) or change-inducing treatments, such as administration of antibiotics, to knock down or alter the composition of the microbiome (Morgun et al. 2015). In addition to transferring whole communities, individual microbial strains that are identified from whole-community profiling can be targeted for isolation (Faith et al. 2010) by using classical microbiology techniques or engineered systems, such as microfluidics. That approach allows the microbial physiology or biochemistry of individual strains of interest (such as secretion products or biomolecular repertoires) to be finely measured and manipulated. Such in vitro systems can be scaled up to include laboratory profiling of entirely synthetic communities, particularly in continuous-culture systems. Detailed properties of host immune sensing and control of a microbiome can be profiled from human tissues (Honda and Littman 2016) by measuring T-cell and B-cell populations, immunoglobulins, cytokine pools, small molecules, and gene expression. The profiling is most often conducted on microbial communities in the gut but can be done for any site-specific community, such as the oral cavity or the skin (Belkaid and Segre 2014). Computational analyses can complement any of the approaches discussed.
 
The first step in a microbiome study typically involves the collection of stabilized microbial biomass specimens that will be used and analyzed in various assays. Each sampling method for human-associated microbial community types has strengths and weaknesses that are driven by the dramatically different microbial ecologies in or on the body. The methods that have been established for gathering a sample of sufficient biomass (referring to the quantity of microorganisms needed for an assay) for each major body site are described here, and limitations of each approach are noted.
 
The gut microbiome is most commonly sampled from stool, which represents well the microbial community of the colonic lumen and to a smaller degree that of the small intestine (Yasuda et al. 2015). Stool is easily obtained for sampling, has extreme microbial density and minimal human genetic contamination (HMP Consortium 2012a,b), and contains material that can be assayed with a variety of molecular techniques. Because microbial characteristics can change rapidly with environmental conditions (such as a sudden decrease in temperature and exposure to air), it is important to take steps to preserve samples by, for example, immediately freezing them or using various laboratory protocols and commercial kits to fix them (Franzosa et al. 2014; Song et al. 2016). It is possible to culture many microorganisms from frozen stool samples, whereas fixatives typically kill microorganisms (preventing culture) and might not be compatible with conducting some molecular assays at a later time. Fixatives do, however, allow convenient collection and shipping of samples. In a clinical setting, mucosal biopsies are common and provide a more precise and biogeographically resolved snapshot of the mucosally associated microbial community (Morgan et al. 2012), but they are more challenging to obtain and can be assayed only with technologies that are not affected by the presence of human cells in the sample. Other sample types, such as mucosal brushing or rectal swabs, are also possible but are less well studied with respect to protocol consistency and community representation (Tong et al. 2014).
 
Animal models are widely used to investigate the human microbiome for several reasons. First, it is much easier to manipulate animal models than human subjects experimentally. Animal studies allow the careful control of experimental variables, scalability, and reproducibility that is often impossible in human studies. Second, ecologic and physiologic attributes of the animal body are highly complex and dynamic and cannot be comprehensively recapitulated in in vitro or in silico models. Finally, the common ancestry of humans and other animals has resulted in the conservation of many genomic, molecular, cellular, and physiologic traits across animal lineages and allows many (not all) findings derived from animal studies to be extrapolated to humans. The advantages of using animal models are counterbalanced by important caveats, including salient differences among animal lineages in anatomy, physiology, and microbiomes (Ley et al. 2008). Although the caveats might limit the relevance of animal models for understanding some aspects of the human microbiome, animal models are important in the larger field of microbiome science.
 
Several fundamental experimental strategies can be used to study microbiomes in animal models. First, animals can be used to test whether the microbiome composition and function correlate with such variables as host age, host genotype, host body site, diet, and chemical or other experimental exposures. The experiments are typically performed on laboratory or wild animals that are colonized by complex microbial communities. Second, animals can be used to study the effects of the presence or composition of a microbiota on host phenotypes. To test whether microbiome composition contributes to host phenotypes, animals with an intact microbiome can be treated with broad-spectrum antibiotics to reduce microbial abundance and alter community composition. That is a relatively inexpensive and rapid way to disrupt the microbiome, but its disadvantage is that it does not distinguish between the effects induced by loss of antibiotic-sensitive microorganisms, by the remaining antibiotic-resistant microorganisms, or directly by the antibiotics (Morgun et al. 2015). Third, another inexpensive and rapid approach for testing the effect of a particular microbial community or strain is to introduce it directly into conventionally reared animals that are already colonized with a microbiota (a probiotic or super-colonization approach). Introduction can also be achieved by co-housing animals that initially contain distinct microbial communities or strains. However, introduction of microorganisms to compete with the pre-existing microbiota and establish stable colonization has had a low success rate and has resulted in considerable variation in experimental outcomes.

The different experimental approaches described above have been used in a broad array of animal species, including mice, zebrafish, fruit flies, and *Caenorhabditis elegans*. Each species has a unique set of characteristics related to its relative size; transparency; microbiome complexity, composition, and function; genetic variance; and evolutionary distance from humans (Leulier et al. 2017). For example, using mice offers some advantages, such as powerful genetic resources that include inbred lines to reduce the effect of genetic variability, an extensive array of knockout strains, and their relatively close evolutionary distance and physiologic similarity to humans. But the disadvantages of using mice include the difficulty of in vivo imaging and the relatively high cost and low scalability of gnotobiotic and conventional husbandry. In contrast, zebrafish have such advantages as facile in vivo imaging owing to their optical transparency, small size that permits genetic and chemical screens, and scalable and inexpensive husbandry requirements that are easily adjusted for gnotobiotic methods. But the disadvantages of using zebrafis